*** nginx.conf	2014-08-24 01:43:06.000000000 +0900
--- nginx.conf.modified	2014-08-24 01:44:53.000000000 +0900
***************
*** 10,17 ****
--- 10,23 ----
      worker_connections  1024;
  }
  
+ env AIPO_PORT_80_TCP_ADDR;
+ env AIPO_PORT_80_TCP_PORT;
  
  http {
+     
+     perl_set $AIPO_PORT_80_TCP_ADDR 'sub { return $ENV{"AIPO_PORT_80_TCP_ADDR"} }';
+     perl_set $AIPO_PORT_80_TCP_PORT 'sub { return $ENV{"AIPO_PORT_80_TCP_PORT"} }';
+     
      include       /etc/nginx/mime.types;
      default_type  application/octet-stream;
  
